app/internal/menstrual_predictor_utils.py

```diff
@@ -0,0 +1,148 @@
+import datetime
+from datetime import timedelta
+from typing import List, Union
+
+from fastapi import Depends
+from loguru import logger
+from sqlalchemy import asc
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+
+from app.database.models import Event, UserMenstrualPeriodLength
+from app.dependencies import get_db
+from app.internal.security.dependencies import current_user
+from app.internal.security.schema import CurrentUser
+from app.routers.event import create_event
+
+MENSTRUAL_PERIOD_CATEGORY_ID = 111
```

Comment:
Define this only in one file, and import it from there. Don't define it multiple times on multiple files.